help/templates/template.cshtml

```diff
@@ -186,6 +186,9 @@
 															<ul>
 																<li><a href="/windows-chocolatey.html">Chocolatey</a></li>
 															</ul>
+                                                            <ul>
+																<li><a href="/fake-windows-registry.html">Registry</a></li>
```

Comment:
Note: I think you want `/apidocs/v5/fake-windows-registry.html` here as we don't have a specific markdown for `fake-windows-registry.md`. Feel free to generate the documentation locally (`fake build target GenerateDocs` and running it `fake build target HostDocs` )

Reply:
Yea you're right, just realized I did that wrong haha